packages/zoe/src/issuerTable.js

```diff
@@ -39,19 +39,26 @@ const makeIssuerTable = () => {
       const brandP = E(issuerP).getBrand();
       const brandIssuerMatchP = E(brandP).isMyIssuer(issuerP);
       const amountMathKindP = E(issuerP).getAmountMathKind();
-      /** @type {[Issuer,Brand,boolean,AmountMathKind]} */
-      const [
-        issuer,
-        brand,
-        brandIssuerMatch,
-        amountMathKind,
-      ] = await Promise.all([
-        issuerP,
-        brandP,
-        brandIssuerMatchP,
-        amountMathKindP,
-      ]);
-      // AWAIT /////
+
+      let issuer;
+      let brand;
+      let brandIssuerMatch;
+      let amountMathKind;
+      try {
+        /** @type {[Issuer,Brand,boolean,AmountMathKind]} */
+        ([issuer, brand, brandIssuerMatch, amountMathKind] = await Promise.all([
```

Comment:
This type declaration isn't actually having the effect it should. If you hover over any of the variables in the next line, they show as `any`.  I'd try putting the type declarations on the variable declarations (lines 43-46)